src/components/Task.js

```diff
@@ -5,7 +5,7 @@ function Task({ text, category, handleDelete }) {
 // console.log(task) 
 
   function handleClick(){
-    handleDelete(task)
+    handleDelete({text, category})
```

Comment:
we get the object as text/category under test, and the tests check `handleDelete`